documentation/universal_links.md

```diff
@@ -0,0 +1,250 @@
+   - Select your app target from the list of targets.
+   - Click on the "Capabilities" tab at the top of the editor.
+
+3. **Enable Associated Domains**:
+
+   - In the Capabilities section, find "Associated Domains" and toggle the switch to enable it.
+   - A new section should appear to configure your associated domains.
+
+4. **Add Your Associated Domains**:
+
+   - Add each domain you want to associate with your app in the format: `applinks:yourdomain.com`
+     - For example, `applinks:example.com` if your domain is `example.com`.
+
+5. **Verify the Entitlements File**:
+
+   - Xcode will automatically add an entitlements file to your project if it doesn't exist, or update your existing one. You can verify this by checking for a file named `YourApp.entitlements` in your project.
+   - Open this file to ensure the associated domains are listed correctly under the `<key>com.apple.developer.associated-domains</key>` entry.
+
+6. **Build and Run Your App**:
+   - Make sure your app's provisioning profile includes support for the associated domains capability.
+   - Build and run your app on a device to test the setup.
+
+### Points to Consider
+
+- **HTTPS Requirement**: Ensure all of your associated domains are served over HTTPS.
+- **Multiple Domains**: You can list multiple associated domains by adding each one on a new line in the Associated Domains section within Xcode.
+- **Testing**: After setting up, test your Universal Links by opening valid URLs on a device to ensure they correctly redirect to your app.
+
+> [!IMPORTANT]
+>
+> `.well-known/apple-app-site-association` files will **not** be served from "\*.myshopify.com" domains.
+> You must connect a non-myshopify domain under https://admin.shopify.com/settings/domains to serve these files.
+
+## Configuring Universal Links for your Storefront
+
+<img alt="iOS Buy SDK Configuration" src="media/ios_configuration.png" width="500" />
+
+- Start by navigating to your storefront's Admin panel at https://admin.shopify.com/settings/apps/development.
+- If an app is already set up, select the appropriate one. If not, proceed to create a new app.
+- Within your app's "Configuration" tab, click "Edit" on the "Storefront API" section.
+- In the "iOS Buy SDK" section, enter your Apple App ID, activate "Use iOS Universal Links," and upload your Apple app certificate.
+
+> [!NOTE]
+> Your Apple app certificate will ensure the integrity of your apple-app-site-association file.
+
+The result of configuring Universal Links for your store will be a `.well-known/apple-app-site-association` JSON file served from your public storefront domain. This file informs iOS about the routes that should open directly in your app instead of in a web browser. Apple will periodically fetch this file to configure app routing on buyer's devices. This process is essential for enabling deep linking and ensuring a seamless app experience when users navigate links associated with your app.
```

Comment:
> Apple will periodically fetch this file to configure app routing on buyer's devices.

More specifically [Apple seems to say its about once a week](https://developer.apple.com/documentation/xcode/supporting-associated-domains), though this isn't really a given, general advice is to issue a new app update in order to get this updated

>  This process is essential for enabling deep linking

`deep linking` should be `universal linking`